eclipse-emfcloud/emfcloud-modelserver
# Fix subscription connections and undo/redo calls in ClientAPI

- Update dependencies

- Fix format parameter, such that subscriptions work correctly again for a requested format (was falsely removed with 96085a0)
- Add missing parameter modelUri to undo/redo calls in ClientAPI
- Add basic client subscription example
- Add ModelServerClientTests for undo, redo and save calls
- Remove warning in SessionControllerTest
- Update README

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

Please also see https://github.com/eclipse-emfcloud/emfcloud-modelserver-theia/pull/27 which adds an example usage of subscribing to a model in a certain format.
